File: src/main/java/cairo/psi/CairoElementType.java
package cairo.psi;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;
import cairo.CairoLanguage;

public class CairoElementType extends IElementType {
	public CairoElementType(@NotNull @NonNls String debugName) {
		super(debugName, CairoLanguage.INSTANCE);
	}
}

File: src/main/kotlin/cairoplugin/language/psi/CairoElementType.kt
package cairoplugin.language.psi

import cairoplugin.language.CairoLanguage
import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import org.jetbrains.annotations.NotNull

class CairoElementType(@NotNull @NonNls debugName: String) : IElementType(debugName, CairoLanguage.INSTANCE) {

}